webapp/src/immersivereader.tsx

```diff
@@ -182,8 +182,7 @@ function getTokenAsync(): Promise<ImmersiveReaderToken> {
                 },
                 e => {
                     pxt.storage.removeLocal(IMMERSIVE_READER_ID);
-                    console.log("immersive reader fetch token error: " + JSON.stringify(e));
-                    pxt.tickEvent("immersiveReader.error", { error: JSON.stringify(e) });
+                    pxt.tickEvent("immersiveReader.error", e);
```

Comment:
I don't think we ever log the entire exception in tickEvent, it will grab the stack trace which might be large.

Reply:
you can call reportException to get the exception details and also call tickEvent to see in aggregate analytics